benchmark/benchmark.rb

```diff
@@ -0,0 +1,30 @@
+require 'webmock'
+require 'devcycle-ruby-server-sdk'
+require 'benchmark'
+
+include WebMock::API
+
+WebMock.enable!
+WebMock.disable_net_connect!
+
+stub_request(:get, 'https://config-cdn.devcycle.com/config/v1/server/dvc_server_token_hash.json').
+  to_return(headers: { 'Etag': 'test' }, body: File.new('../examples/local/local-bucketing-example/test_data/large_config.json'), status: 200)
```

Comment:
use __dir__ to get the proper relative path too - same way we loaded WASM.

Reply:
idk if I can use `__dir__` here cause we need to go up a directory and then to the config file. I can make a copy of it the file and move it in the benchmark folder but I'd rather not have a duplicate